templates/logosSection.hbs

```diff
@@ -0,0 +1,16 @@
+<section class="aliados-wrapper">

```

Comment:
Same recommendation about the name in english. Also is a good practice to be consistent with the names across files. So, I would suggest to rename the file to sponsorsSection.hbs